tmt/steps/provision/podman.py

```diff
@@ -152,19 +152,25 @@ def podman(self, command, **kwargs):
         """ Run given command via podman """
         return self.run(['podman'] + command, shell=False, **kwargs)
 
-    def execute(self, command, environment=None):
+    def execute(self, command, **kwargs):
         """ Execute given commands in podman via shell """
         if not self.container:
             raise tmt.utils.ProvisionError(
                 'Could not execute without provisioned container.')
 
+        # Change to given directory on guest if cwd provided
+        directory = kwargs.get('cwd', '')
+        if directory:
+            directory = f"cd '{directory}'; "
+
         # Note that we MUST run commands via bash, so variables
         # work as expected
         if isinstance(command, list):
             command = ' '.join(command)
+        command = directory + command
```

Comment:
I am afraid this won't work:

```python
subprocess.check_call(['podman', 'run', 'fedora', 'cd /etc; cat passwd'])                                                                             
Error: container_linux.go:346: starting container process caused "exec: \"cd /etc; cat passwd\": stat cd /etc; cat passwd: no such file or directory": OCI runtime command not found error
xonsh: For full traceback set: $XONSH_SHOW_TRACEBACK = True
subprocess.CalledProcessError: Command '['podman', 'run', 'fedora', 'cd /etc; cat passwd']' returned non-zero exit status 127.
```
or am I missing something?

Reply:
note that for ssh it works:

```
$ ssh mvadkert@freedom.usersys.redhat.com 'cd /etc; head -1 /etc/group'
root:x:0:
```

Reply:
I guess the solution could be pass the command to `sh -c` explicitly ... note sure about a better way now
```
subprocess.check_call(['podman', 'run', 'fedora', 'sh', '-c', 'cd /etc; head -1 passwd'])                                                                                                                  
root:x:0:0:root:/root:/bin/bash
```

Reply:
This is working because the command is converted to string a couple of lines above. The current working directory is used by the `run.sh` executor so it's also verified by it:

```
> tmt run -adddvvv plan --name smoke provision -h container
...
execute
    workdir: /var/tmp/tmt/run-641/plans/smoke/execute
    how: shell
    order: 50
        Copy '/home/psss/git/tmt/tmt/steps/execute/run.sh' to '/var/tmp/tmt/run-641/plans/smoke/execute'.
        Run command 'podman exec tmt-var-tmp-tmt-run-641-plans-smoke sh -c cd '/var/tmp/tmt/run-641/plans/smoke/execute'; ./run.sh -v .. shell stdout.log stderr.log'.
    Read file '/var/tmp/tmt/run-641/plans/smoke/execute/stdout.log'.
    stdout.log: .D
```

Do you have any other concerns?